ndex2/client.py

```diff
@@ -878,11 +1211,29 @@ def set_network_system_properties(self, network_id, network_properties):
 
     def update_network_profile(self, network_id, network_profile):
         """
+
         Updates the network profile
         Any profile attributes specified will be updated but attributes that are not specified will
         have no effect - omission of an attribute does not mean deletion of that attribute.
         The network profile attributes that can be updated by this method are: 'name', 'description' and 'version'.
 
+        .. code-block:: python
+
+		import ndex2.client as nc
+		import ndex2
+		import networkx as nx
+		import pandas as pd
+		import os
+
+		#note: cannot use this method to update visibility，
+		#Please use make_network_public/private function to set network visibility.
+		#Also can use set_network_properties() to set visibility
+		network_id = "INSERT NETWORK UUID HERE"
+		network_profile={"name":"My Network", 
+				 "description":"learn from Tutorial"}
+		my_ndex.update_network_profile(network_id, network_profile)   
```

Comment:
Please add line showing how `my_ndex` is created. The only import needed is the one to import the Ndex2 client